src/main/java/it/gov/pagopa/reportingorgsenrollement/Application.java

```diff
@@ -1,7 +1,20 @@
 package it.gov.pagopa.reportingorgsenrollement;
 
 import io.micronaut.runtime.Micronaut;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
 
+@OpenAPIDefinition(
+        info = @Info(
+                title = "Reporting Organizations Enrollments",
```

Comment:
lift & shift this info into [configuration file](https://micronaut-projects.github.io/micronaut-openapi/latest/guide/#propertiesFileConfiguration)